src/containers/Room.tsx

```diff
@@ -152,7 +152,9 @@ export const Room = ({ roomId, refetchIfNeeded, refetchRequested, hidden }: Room
         <div className="flex flex-col w-150 gap-2">
           <AddRtspComponent roomId={roomId} refetchIfNeeded={refetchIfNeededInner} />
 
-          <AddHlsComponent roomId={roomId} refetchIfNeeded={refetchIfNeededInner} />
+          {room.roomStatus.config.videoCodec === "h264" && (
+            <AddHlsComponent roomId={roomId} refetchIfNeeded={refetchIfNeededInner} />
+          )}
```

Comment:
I'm not sure if we should simply hide it without providing any information to the user.

Reply:
That's true. I'll change it to disable the button with information about the codec